substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/LLVMNativeImageCodeCache.java

```diff
@@ -196,10 +196,11 @@ private LLVMStackMapInfo readStackMapSection(LLVMSectionIteratorRef sectionItera
 
     private void storeMethodOffsets(long codeSize) throws IOException {
         List<Integer> sortedMethodOffsets = textSymbolOffsets.values().stream().distinct().sorted().collect(Collectors.toList());
-        Integer gcRegisterOffset = textSymbolOffsets.get(SYMBOL_PREFIX + "__svm_gc_register");
-        if (gcRegisterOffset != null) {
-            sortedMethodOffsets.remove(gcRegisterOffset);
-        }
+        textSymbolOffsets.forEach((symbol, offset) -> {
+            if (symbol.equals("__svm_gc_register") || symbol.startsWith(SYMBOL_PREFIX + "__svm_jni_wrapper_")) {
```

Comment:
These should become constants as they are used in multiple cases.

Reply:
done